src/resources/views/crud/form_content.blade.php

```diff
@@ -12,6 +12,10 @@
   </div>
 @endif
 
+@php
+    session(['current_crud_loaded_fields' => $crud->getLoadedFieldTypes()]);
```

Comment:
When parent crud, let's say `ArticleCrud`, loads it's create/update form we save to session what fields are loaded. When loading the related entity in the modal to create, let's say `CategoryCrud`, when we fetch the the modal we get the session value and mark the `ArticleCrud` fields as loaded in the `CategoryCrud` modal so we don't load the same assets that are already on page. 

From `on_the_fly_modal.blade.php` : 

```php
if (session()->has('current_crud_loaded_fields')) {
        $loadedFields = session('current_crud_loaded_fields');
        session()->forget('current_crud_loaded_fields');
    }

    $loadedFields = $loadedFields ?? [];

    //mark parent crud fields as loaded in DOM.
    foreach($loadedFields as $loadedField) {
        $crud->markFieldTypeAsLoaded($loadedField);
    }
```

Maybe this can be replaced with the `BareCrud` functionality that we implemented. Get the controller instance and the fields. Problem is that in this `BareCrud` we have no request, session or whatever, and calling the controller `setupCreateOperation()` might break things when developer used any of those functionalities to setup his create operation.

Reply:
Done :+1: